Library/Homebrew/formula_auditor.rb

```diff
@@ -590,6 +590,28 @@ def audit_bottle_spec
       new_formula_problem "New formulae in homebrew/core should not have a `bottle do` block"
     end
 
+    def audit_eol
+      return if formula.deprecated? || formula.disabled?
```

Comment:
```suggestion
      return if formula.deprecated? || formula.disabled?
      return if @core_tap
```

given different taps may have different policies (e.g. taps that do `brew extract`).

Reply:
@SMillerDev I think you lost this change on last push